src/main/java/com/shaft/dsl/gui/Button.java

```diff
@@ -11,11 +12,14 @@ public Button(WebDriver driver, By locator) {
     public void click() {
         elementActions.click(locator);
     }
-    public String getText() {
-        return elementActions.getText(locator);
-    }
     public Boolean isEnabled(){
         return Boolean.parseBoolean(elementActions.getAttribute(locator,"disabled"));
     }
+    public void shouldBeEnabled()
+    {  Validations.assertThat().object(isEnabled()).isTrue().perform();
```

Comment:
we need to use assertThat().element().attribute() in this one instead of assert true/false.
the element attribute assertion can capture screenshot evidence and highlight the element in green or yellow if it passed or failed respectively.